sql-plugin/src/main/scala/org/apache/spark/sql/rapids/ExternalSource.scala

```diff
@@ -121,6 +129,10 @@ object ExternalSource {
   }
 
   def getScans: Map[Class[_ <: Scan], ScanRule[_ <: Scan]] = {
+    if (showErrorMessage) {
+      println("WARNING: Avro library isn't loaded by the RAPIDS plugin. Please use --jars to " +
+          "load the plugin if you haven't done so")
```

Comment:
Since this is optional, I think the warning could be worded better.
Maybe something like: "WARNING: Avro library not found by the RAPIDS plugin.  If needed, please use --jars to add it"
Should we include the full classpath in the error message?

Reply:
I can change that